zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/CassandraSpanConsumer.java

```diff
@@ -18,8 +18,10 @@
 import com.datastax.driver.core.utils.UUIDs;
 import com.google.common.base.Preconditions;
 import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
always linkedhashmap to prevent test flakes on order

Reply:
Unused import